index.html

```diff
@@ -315,6 +315,62 @@ <h2 class="connect">LET'S CONNECT</h2>
           </div>
         </div>
       </section>
+
+      <section class="footer-container"> 
+        <footer class="footer-section">
+          <div class="contact-me-form">
+            <div class="contact-info">
+              <h2 class="contact-title">Contact me</h2>
+              <p class="content-text">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d love
+                to help with it
+              </p>
+            </div>
+  
+            <form action="https://formspree.io/f/xpznvyer" method="POST"      class="contact-form">
+              <div class="footer-field">
+                <input
+                  required
+                  id="name"
+                  name="user-name"
+                  placeholder="Yeremias"
+                  minlength="1"
+                  maxlength="30"
+                  type="text"
+                />
+              </div>
+  
+              <div class="footer-field">
+                <input required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" name="Email" id="email" placeholder="natajayanj14@gmail.com"
+                 type="email" />
+              </div>
+  
+              <div class="footer-field">
+                <textarea
+                  name="message"
+                  id="message"
+                  minlength="1"
+                  placeholder="Write your message here"
+                  maxlength="500"
+                  required
+    
+                  type="text"
+                  cols="30"
+                  rows="10"
+                ></textarea>
+              </div>
+  
```

Comment:
- [ ] kindly fix those design issues to match Figma template : 

- Kindly Remove the red outline/border from your form 
- Kindly remove the empty space from the left side


![image](https://user-images.githubusercontent.com/89282221/207367844-e7d084b8-0788-49a7-b3e4-92ac01eff263.png)